From: Simon McVittie <smcv@collabora.com>
Date: Wed, 14 Feb 2018 14:51:19 +0000
Subject: Remove unused FUSE build-dependency

This was needed for the document portal, which has been removed.

Signed-off-by: Simon McVittie <smcv@collabora.com>
Forwarded: https://github.com/flatpak/flatpak/pull/1417
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index c1427d2..601fbd2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -222,8 +222,6 @@ fi
 
 PKG_CHECK_MODULES(OSTREE, [ostree-1 >= $OSTREE_REQS])
 
-PKG_CHECK_MODULES(FUSE, [fuse])
-
 PKG_CHECK_MODULES(JSON, [json-glib-1.0])
 
 PKG_CHECK_MODULES(APPSTREAM_GLIB, [appstream-glib >= 0.5.10])
